### PR TITLE
[uxd-615]index page modifacation

### PR DESCRIPTION
**Pull request by ThyeeZz**

### CODE REVIEW

File: src/components/docLayout/index.jsx [hunk 1]
```diff
@@ -163,25 +165,22 @@ const DocLayout = props => {
       <NewHeader
         locale={locale}
         type="doc"
-        isSecondHeader={true}
-        className={styles.header}
+        isSecondHeader={false}
```
Comment (Tumao727) [inline src/components/docLayout/index.jsx:168]: delete it

File: src/components/card/startCard/startCard.jsx [hunk 2]
```diff
@@ -22,12 +25,23 @@ const StartCard = ({ data, wrapperClass = '' }) => {
...
+        {
+          link === 'bootcamp' ? (
+            <Link to='/bootcamp' className={styles.btn}>
+              <span className={styles.text}>{isMobile ? 'Learn' : btnLabel}</span>
+              <span className={styles.icon}>
+                <i className="fa fa-chevron-right"></i>
+              </span>
+            </Link>
+          ) : (
+            <a href={link} className={styles.btn}>
```
Comment (nameczz) [inline src/components/card/startCard/startCard.jsx:37]: component is same.. just link need change

File: src/hooks/index.jsx [hunk 3]
```diff
@@ -25,6 +25,8 @@ export const useClickOutside = (ref, handler, events) => {
 
   useEffect(() => {
     const detectClickOutside = event => {
+      console.log(event.target);
+      console.log(ref.current);
```
Comment (nameczz) [inline src/hooks/index.jsx:29]: delete console

File: src/components/header/v2/index.jsx [hunk 4]
```diff
@@ -130,17 +117,24 @@ const V2Header = ({
                       target="_blank"
                       rel="noreferrer"
                     >
-                      {label}
+                      {
```
Comment (Tumao727) [inline src/components/header/v2/index.jsx:120]: not use icon as condition, accept component to be label type

File: src/pages/index.jsx [hunk 5]
```diff
@@ -69,7 +69,7 @@ const HomePage = ({ data, pageContext }) => {
                   link={banner.startBtn.href}
                   children={
                     <>
-                      <span>{banner.startBtn.label}</span>
+                      <span>{banner.startBtn.label.toUpperCase()}</span>
```
Comment (nameczz) [inline src/pages/index.jsx:72]: use css to control upper case

File: src/pages/index.jsx [hunk 6]
```diff
@@ -81,21 +81,18 @@ const HomePage = ({ data, pageContext }) => {
...
-                className="banner-right-anime"
-              />
-            </div>
-
-            <div className="banner-bg-container"></div>
+            <div className="banner-right"></div>
+          </div>
+          <div className="slogan">
+            <p className='text'>We have literally been unable to kill this thing. No matter what we’ve thrown at it.</p>
+            <p className="name">by Mi.Alato</p>
```
Comment (nameczz) [inline src/pages/index.jsx:95]: what's this ? do we need i18n?

File: src/templates/community.jsx [hunk 7]
```diff
@@ -9,6 +9,7 @@ import mailIcon from '../images/community/mail.png';
 import { useCodeCopy, useFilter } from '../hooks/doc-dom-operation';
 
 export default function CommunityTemplate({ data, pageContext }) {
+  console.log(pageContext);
```
Comment (nameczz) [inline src/templates/community.jsx:12]: delete  console

**Review by Tumao727 — APPROVED** [review]

**Pull request merged**